homeassistant/components/surepetcare/__init__.py

```diff
@@ -139,24 +137,26 @@ def __init__(self, hass: HomeAssistant, surepy: Surepy) -> None:
         """Initialize the Sure Petcare object."""
         self.hass = hass
         self.surepy = surepy
-        self.states = {}
+        self.states: dict[int, Any] = {}
 
     async def async_update(self, _: Any = None) -> None:
         """Get the latest data from Sure Petcare."""
 
+        _LOGGER.debug("🐾  updating sure petcare data...")
+
         try:
-            self.states = await self.surepy.get_entities()
+            self.states = await self.surepy.get_entities(refresh=True)
```

Comment:
Why is `refresh=True` needed?

Reply:
this enforces to call the API and do not use some cached data

Reply:
Yes, but is that needed? And why has it not been needed before?

Reply:
Are you sure it isn't at the moment? :D

Reply:
?

Reply:
Please do not mark the conversation as resolved without addressing the comment.
Could you explain why you add `refresh=True` ?

Reply:
Could you explain why you think that isn't needed? ...on a day where you closed several issues about not updating entities..?!

As I explained few lines above...
> this enforces to call the API and not use some cached data

But hey, I don't care. I will remove it okay?

Reply:
I do not know if it is needed or not.
I would like to know why it is needed? And why has it not been needed before? 

It will double the number of requests to the surepetcare servers? So we should not do it if not needed

Reply:
what? why double the requests?!

https://github.com/benleb/surepy/blob/1d684016dc6d31747b73f0bc4c7bd520258549ac/surepy/__init__.py#L322

That's the line in the lib, maybe it's easier if you have a look yourself...

Reply:
Ok, do not fully understand it.
If you are you are sure, it is good enough for me :)